components/skill/SkillPage.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import {
+  FormControl,
+  Input,
+  InputLabel,
+} from "@mui/material";
+import { Skill } from "../../types/hero";
+import { SkillSection } from "./SkillSection";
+import { BoundedColumn } from "../layout/BoundedColumn";
+
+export function SkillPage({ skills }: { skills: Skill[] }) {
+  const [searchText, setSearchText] = useState("");
+
+
+  // Filter skills based on the search text in both name and description
+  const filteredSkills = skills.filter(skill =>
+     skill.name.toLowerCase().includes(searchText.toLowerCase()) ||
+     (skill.description && skill.description.toLowerCase().includes(searchText.toLowerCase()))
+  );
+
+  return (
+    <div className="bg-white flex flex-grow justify-center flex-col cursor-auto">
+      <h1 className="text-6xl text-center mt-5 mb-10 font-thin text-gray-600">
+        Skills
+      </h1>
+      <div className="flex flex-wrap justify-center text-center mb-5">
+        <div className="mr-4">
+          <FormControl>
+            <InputLabel>Search</InputLabel>
+            <Input
+              value={searchText}
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                setSearchText(event.target.value);
+              }}
+            />
+          </FormControl>
+        </div>
+      </div>
+
+      {/* Skills list */}
+      <BoundedColumn>
```

Comment:
to make this center properly it needs a parent that is w-100

![image](https://github.com/RedWrath5/wikigrisser-next/assets/6718748/34228423-e1a9-425b-ab7e-974308aca475)

Reply:
Fixed
![image](https://github.com/RedWrath5/wikigrisser-next/assets/34580058/46590165-e4cf-4d18-876e-15a3b89805a6)